tests/virtualization/yast_virtualization.pm

```diff
@@ -8,15 +8,22 @@ sub run() {
     wait_idle;
     send_key "alt-f10";
     become_root;
-    script_run "yast2 virtualization";
-    wait_idle;
-    assert_screen "virt-sle12sp1-gnome_yast_virtualization", 50;
-    # select everything
-    send_key "alt-x", 10;    # XEN Server
-    send_key "alt-e", 10;    # Xen tools
-    send_key "alt-k", 10;    # KVM Server
-    send_key "alt-v", 10;    # KVM tools
-    send_key "alt-l", 10;    # libvirt-lxc
+    script_run("/sbin/yast2 virtualization; echo yast2-virtualization-done-\$? > /dev/$serialdev", 0);
+    if (check_var("FLAVOR", "Desktop-DVD")) {
+        assert_screen "virt-sled12sp2-gnome_yast_virtualization";
```

Comment:
Hi coolo,

- The test point of this case is install Hypervisors (including Server & Tools) on SLE. But on SLE Desktop,we do not provide host qemu packages, so step 6 is different between SLES and SLED.

    SLES:http://147.2.212.239/tests/351/modules/yast_virtualization/steps/6/edit

    SLED:http://147.2.212.239/tests/424/modules/yast_virtualization/steps/6/edit

    And we don't want create a new virtualization test suite which contains only 2 cases for SLED, so I added some check_var to make this case can handle two scenarios. What do you think?

- As for the tag name, I don't want to change the original style -_-||, but I will correct it.